src/lambda_codebase/account_processing/create_account.py

```diff
@@ -42,4 +48,5 @@ def create_account(account, adf_role_name, org_client):
 
 def lambda_handler(event, _):
     org_client = boto3.client("organizations")
-    return create_account(event, ADF_ROLE_NAME, org_client)
+    details = create_account(event, ADF_ROLE_NAME, org_client)
+    EVENTS.put_event(detail=json.dumps(details), detailType="ACCOUNT_PROVISIONED", resources=[details.get("account_id")])
```

Comment:
Please reformat this. https://black.readthedocs.io/en/stable/getting_started.html can automatically do this for you